lib/opFns.js

```diff
@@ -318,6 +318,22 @@ module.exports = {
       cb(err)
     })
   },
+  RETURNDATASIZE: function (runState) {
+    return utils.intToBuffer(runState.lastReturned.length)
+  },
+  RETURNDATACOPY: function (memOffset, returnDataOffset, length, runState) {
+    memOffset = utils.bufferToInt(memOffset)
+    returnDataOffset = utils.bufferToInt(returnDataOffset)
+    length = utils.bufferToInt(length)
+    
+    if (returnDataOffset + length > runState.lastReturned.length) {
+      trap(ERROR.OUT_OF_GAS)
```

Comment:
@gasolin I just took this from the EIP specification, there ``out-of-gas`` is mentioned: https://github.com/ethereum/EIPs/pull/211/files